production_api/preprocess_data.py

```diff
@@ -0,0 +1,57 @@
+import re
+
+
+class DataPreprocessor:
+    """Class to preprocess the description
+    
+    Returns:
+        string -- the preprocessed text
+    """
+
+    def __init__(self):
+        self.keywords = [
```

Comment:
This could also be filled by a keyword param with ["email", "fax", ...] as the default or base list. The name `keywords` is a bit misleading here since you treat these as stopwords. I understand they don't not really fit into the profile of usual stop word candidates either.